src/ep.config.json

```diff
@@ -27,12 +25,12 @@
     }
   ],
   "defaultCurrencyValue": "CAD",
-  "skuImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%sku%.png",
-  "siteImagesUrl": "https://elasticpath-demo-images.s3-us-west-2.amazonaws.com/VESTRI_VIRTUAL_V1/%fileName%",
+  "skuImagesUrl": "https://refstoreimgs.s3-us-west-2.amazonaws.com/productImages/%fileName%",
+  "siteImagesUrl": "https://refstoreimgs.s3-us-west-2.amazonaws.com/siteImages/%fileName%",
   "enableOfflineMode": false,
   "imageFileTypes": {
     "enable": true,
-    "types": [".png" ,".jpg", ".svg"]
```

Comment:
Top element named imageFileTypes but no control of types. I suggest we add back the types to give devs the options to set new images.

Reply:
WIP

Reply:
updated